src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -44,8 +46,11 @@ public class ElevatorSubsystem extends SubsystemBase {
   private final MotionMagicVoltage m_request = new MotionMagicVoltage(0);
 
   // motors
-  private TalonFX m_motor;
-  private TalonFX m_motor2;
+  private final TalonFX m_motor;
+  private final TalonFX m_motor2;
```

Comment:
Can we name these something more clear?  LIke m_motor_left and m_motor_right?  Some way that people will know which is which on the bot.